workflow/Snakefile_working

```diff
@@ -199,3 +194,24 @@ rule make_all_predictions:
 		done
 		gzip {params.output_dir}/AllPredictions.txt
 		"""
+
+rule gen_qc_plots:
+	input: 
+		candidateRegions = os.path.join(config['predictions_results_dir'], "{biosample}", "Peaks", "macs2_peaks.narrowPeak.sorted.candidateRegions.bed"),
+		neighborhoodDirectory = directory(os.path.join(config['predictions_results_dir'], "{biosample}", "Neighborhoods")),
+		enhPredictionsFull = os.path.join(config['predictions_results_dir'], "{biosample}", "Predictions", "EnhancerPredictionsFull.txt")
+	params:
+		code_dir = config['code_dir'],
+		output_dir = os.path.join(config['predictions_results_dir'], "{biosample}", "Metrics")
+	output:
+		metricsDirectory = directory(os.path.join(config['predictions_results_dir'], "{biosample}", "Metrics"))
```

Comment:
note to self: 
I should probably put specific files? Not sure if Snakemake won't execute this rule if the directory already exists